tests/test_results.py

```diff
@@ -535,6 +536,35 @@ def test_redis_get_result_still_return_result_if_forget_fails(redis_result_backe
         assert mock_client.pipeline.call_count == 1
 
 
+@pytest.mark.parametrize("forget", [True, False])
+def test_redis_get_results_with_forget(
+    rabbitmq_broker: RabbitmqBroker,
+    rabbitmq_worker: Worker,
+    redis_result_backend,
+    forget: bool,
+):
+    rabbitmq_broker.add_middleware(Results(backend=redis_result_backend))
+
+    # And an actor that stores a result
+    @remoulade.actor(store_results=True)
```

Comment:
I don't think you're testing what you want to test. I think you need to reimplement get_result in redis backend instead.